storybookjs/storybook
# Build: Update example to Angular 10

Issue: na

## What I did
Update angular-cli exemple to angular 10

## How to test

- Is this testable with Jest or Chromatic screenshots?
> na
- Does this need a new example in the kitchen sink apps?
> na
- Does this need an update to the documentation?
> na

If your answer is yes to any of these, please make sure to include it in your PR.
